src/service/rankService.ts

```diff
@@ -0,0 +1,41 @@
+export class RankService {
+
+    constructor(private httpClient) {
+    }
+
+    public rankIssue(id, referenceId, direction: string): void {
+        var requestBody = RankService.createRequestJson(id, referenceId, direction);
+
+        this.httpClient.put({url: 'rest/agile/1.0/issue/rank', json: requestBody},
+            (error, response, body) => {
+                if (response.statusCode !== 204) {
+                    console.log(`rest/agile/1.0/issue/${id}/rank ` + response.statusCode + " " + error);
```

Comment:
Instead of doing sth like this, it is good to add a service responsible for collecting information about errors. e.g. Rollbar. I think that it is enough at the beginning because it contains 5000 'error' events durign 1 month.